components/eam/src/chemistry/mozart/lin_strat_chem.F90

```diff
@@ -188,6 +188,18 @@ subroutine lin_strat_chem_inti(phys_state)
 !     
 !   initialize the linoz data
 
+    !!set true for linoz_v3 interpolation in interpolation
+    if (linoz_v3) file%linoz_v3=.true.
+    if (linoz_v2) file%linoz_v2=.true.
+    !!
+    if (linoz_v2.eq..true. .and.linoz_v3.eq..true. ) then
+            write(iulog,*) subname // ', linoz: Both linoz_v2 and linoz_v3 are true, please check'
```

Comment:
Is it possible to have both true? Can you give an example?

Reply:
This is only for pre-caution, I don't think it would be true.